### PR TITLE
[Event] Add new event type

### PR DESCRIPTION
**Pull request by gichan-jang**
Add new event type for new connection.

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [*]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

**Review by songgot — APPROVED** [review]
LGTM :+1:

File: include/nnstreamer-edge-event.h [hunk 1]
```diff
@@ -30,6 +30,7 @@ typedef enum {
   NNS_EDGE_EVENT_NEW_DATA_RECEIVED,
   NNS_EDGE_EVENT_CALLBACK_RELEASED,
   NNS_EDGE_EVENT_CONNECTION_CLOSED,
+  NNS_EDGE_EVENT_CONNECTION_COMPLETED,
```
Comment (songgot) [inline include/nnstreamer-edge-event.h:33]: Does this message indicate that the network connection was successfully (completed)established? Or does it mean that the network connection is active?

Comment (gichan-jang) [reply include/nnstreamer-edge-event.h:33]: It means  that the network connection was successfully established.
If network connection status is changed to inactive, `NNS_EDGE_EVENT_CONNECTION_CLOSED` event will be invoked.

Comment (songgot) [reply include/nnstreamer-edge-event.h:33]: Thank you for the detailed explanation. :pray:

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**